doc/nrf/ug_dev_model.rst

```diff
@@ -357,15 +357,68 @@ This workflow is particularly beneficial if your application is split among mult
 
 In order to implement this approach you first need to create a manifest repository of your own, which just means a repository that contains a :file:`west.yml` manifest file in its root.
 Next you must populate the manifest file with the list of repositories and their revisions.
-In general, the easiest thing to do is to take the :file:`west.yml` in `fw-nrfconnect-nrf`_ and copy its entries directly as a starting point.
+
+In general, the easiest thing to do is to import the :file:`west.yml` in `fw-nrfconnect-nrf`_ using west's manifest imports feature, like this:
+
+.. code-block:: yaml
+
+   # Example application-specific west.yml, using manifest imports.
+   manifest:
+     remotes:
+       - name: ncs
+         url-base: https://github.com/NordicPlayground
+     projects:
+       - name: nrf
+         remote: ncs
+         revision: v1.2.0
+         import: true
+     self:
+       path: application
+
 Then, make the following changes:
 
-  * Add an entry for `fw-nrfconnect-nrf`_ or a forked version of it, if applicable.
-  * Point the entries of any |NCS| repositories that you have forked to your fork and fork revision.
-  * Add any entries for repositories that you need and that are not part of the |NCS|.
+  * Point the entries of any |NCS| repositories that you have forked to your fork and fork revision, by adding them to the ``projects`` list using a new remote.
+  * Add any entries for repositories that you need that are not part of the |NCS|.
+
+For example:
 
-Once you have your new manifest repository, you can use it with west just like you would use `fw-nrfconnect-nrf`_ when :ref:`getting <dm-wf-get-ncs>` and later :ref:`updating <dm-wf-update-ncs>` the source code.
-You just need to replace ``fw-nrfconnect-nrf`` and ``nrf`` with whatever repository name and path you have chosen for your manifest repository.
+.. code-block:: yaml
+
+   # Example your-application/west.yml, using manifest imports, with
+   # an NCS fork and a separate module
+   manifest:
+     remotes:
+       - name: ncs
+         url-base: https://github.com/NordicPlayground
+       - name: your-remote
+         url-base: https://github.com/your-name
+     projects:
+       - name: nrf
+         remote: ncs
+         revision: v1.2.0
+         import: true
+       - name: nrfx
```

Comment:
nrfx is perhaps not an ideal example, because it's part of `nordic_hal`. If you want to explain how to override an existing repo that is part of NCS' manifest, then you should probably explain it a bit further. Otherwise, I think you are better off just using a 3rd-party repo that is not included in NCS as an example, like for example [nanopb](https://github.com/nanopb/nanopb)

Reply:
There are two things going on here from the previous paragraph:

```
Then, make the following changes:

  * Point the entries of any |NCS| repositories that you have forked to your fork and fork revision, by adding them to the ``projects`` list using a new remote.
  * Add any entries for repositories that you need that are not part of the |NCS|.
```

The nrfx project was meant as an example of the first one. I will replace with mcuboot instead and explain further with YAML comments.

Reply:
Right, but nrfx is not an NCS repository. Instead, it comes with `hal_nordic` right?
mcuboot seems like a good candidate instead.